mainwindow.ui

```diff
@@ -408,6 +408,19 @@
              </property>
             </widget>
            </item>
+           <item row="8" column="0">
+            <widget class="QLabel" name="optimiseLabel">
+             <property name="toolTip">
+              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;--optimise&lt;/span&gt;&lt;/p&gt;&lt;p&gt;optimise output gcode, reducing its output size up to 95%, while accepting a very little loss of precision&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>

```

Comment:
Where did you get this text? It looks very similar to the help text except it is much higher a percentage, right?

Reply:
Its the original from the repo:
https://github.com/pcb2gcode/pcb2gcodeGUI/search?q=optimise+output+gcode%2C+reducing+its+output+size+up+to+95%25%2C+while+accepting+a+very+little+loss+of+precision&unscoped_q=optimise+output+gcode%2C+reducing+its+output+size+up+to+95%25%2C+while+accepting+a+very+little+loss+of+precision

I'll change it to 40 now